packages/table/test/table.test.ts

```diff
@@ -0,0 +1,1043 @@
+                <sp-table-body style="height: 120px">
+                    <sp-table-row value="row1" class="row1">
+                        <sp-table-cell>Row Item Alpha</sp-table-cell>
+                        <sp-table-cell>Row Item Alpha</sp-table-cell>
+                        <sp-table-cell>Row Item Alpha</sp-table-cell>
+                    </sp-table-row>
+                    <sp-table-row value="row2" class="row2">
+                        <sp-table-cell>Row Item Bravo</sp-table-cell>
+                        <sp-table-cell>Row Item Bravo</sp-table-cell>
+                        <sp-table-cell>Row Item Bravo</sp-table-cell>
+                    </sp-table-row>
+                    <sp-table-row value="row3" class="row3">
+                        <sp-table-cell>Row Item Charlie</sp-table-cell>
+                        <sp-table-cell>Row Item Charlie</sp-table-cell>
+                        <sp-table-cell>Row Item Charlie</sp-table-cell>
+                    </sp-table-row>
+                </sp-table-body>
+            </sp-table>
+        `);
+        await elementUpdated(el);
+
+        expect(el.selected.length).to.equal(2);
+
+        const rowThree = el.querySelector('.row3') as TableRow;
+        const rowTwo = el.querySelector('.row2') as TableRow;
+        const rowOne = el.querySelector('.row1') as TableRow;
+        const rowOneCheckbox = rowOne.querySelector('sp-table-checkbox-cell');
+
+        expect(rowThree.selected, 'third row selected').to.be.true;
+        expect(rowTwo.selected, 'second row selected').to.be.true;
+        expect(rowOne.selected, 'first row selected').to.be.false;
+        expect(rowOneCheckbox).to.be.null;
+    });
+
+    it('selects user-passed values for .selected array with no [selects="multiple"] specified on Virtualized `<sp-table>`, but does not allow interaction afterwards', async () => {
+        const virtualItems = makeItemsTwo(5);
+
+        const el = await fixture<Table>(html`
+            <sp-table
+                .selected=${['0', '4']}
+                style="height: 120px"
+                .items=${virtualItems}
+                .renderItem=${renderItem}
+                scroller?="true"
+            >
+                <sp-table-head>
+                    <sp-table-head-cell>Column Title</sp-table-head-cell>
+                    <sp-table-head-cell>Column Title</sp-table-head-cell>
+                    <sp-table-head-cell>Column Title</sp-table-head-cell>
+                </sp-table-head>
+            </sp-table>
+        `);
+
+        await oneEvent(el, 'rangeChanged');
+        await elementUpdated(el);
+        expect(el.selected.length).to.equal(2);
+
+        const rowOneItem = el.querySelector('[value="0"]') as TableRow;
+        const rowFiveItem = el.querySelector('[value="4"]') as TableRow;
+
+        expect(rowOneItem.value).to.equal('0');
+        expect(rowFiveItem.value).to.equal('4');
+    });
+
+    it('selects via `click` while [selects="single"]', async () => {
```

Comment:
Oh, here it is. Having `[selects=single]` all together might be helpful for coming back and understanding the test coverage herein.